lib/dradis/plugins/nexpose/field_processor.rb

```diff
@@ -85,5 +87,20 @@ def format_array_as_table(array)
       end
       rows.join("\n")
     end
+
+    def instantiate_evidence(data)
```

Comment:
@dormi I added a `Nexpose::Test` class but it evaluates to a `Hash` and not of type `Nexpose::Test`. This is because the change propagates to more changes to the `FieldProcessor` logic. Let me know if this is too "hackish".